paramiko/sftp_client.py

```diff
@@ -699,7 +715,7 @@ def getfo(self, remotepath, fl, callback=None):
                 reader=fr, writer=fl, file_size=file_size, callback=callback
             )
 
-        return size
+        return file_size
```

Comment:
... in fact it's just extraneous, the real return is above in the `with:` context. if that fails, an exception will be thrown. this return will never be hit

(but it's fine to just appease flake8 for now)